tests/unit/amo/sagas/test_home.js

```diff
@@ -66,7 +69,7 @@ describe(__filename, () => {
 
       const baseArgs = { api: state.api };
       const baseFilters = {
-        page_size: LANDING_PAGE_ADDON_COUNT,
+        page_size: LANDING_PAGE_EXTENSION_COUNT,
```

Comment:
Because we are expecting different results for different API calls, I don't think we should keep `baseFilters`. We should just specify exactly what is expected in each case (like you've done for the themes case).